src/server/game/Handlers/MiscHandler.cpp

```diff
@@ -1786,7 +1786,7 @@ void WorldSession::HandleMoveSetCanFlyAckOpcode(WorldPacket & recv_data)
     recv_data.readPackGUID(guid);
 
     // pussywizard: typical check for incomming movement packets
-    if (!_player->m_mover || !_player->m_mover->IsInWorld() || _player->m_mover->IsDuringRemoveFromWorld() || guid != _player->m_mover->GetGUID())
+    if (!_player)
```

Comment:
please remove the above comment `// pussywizard: typical check for incomming movement packets`

if it  makes no longer sense

Reply:
okay